src/index.css

```diff
@@ -1,5 +1,5 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+body {
```

Comment:
We are using tailwind.css please compare with dev branch in src/index.css. it needs to have the tailwind.css